cluster/images/etcd/OWNERS

```diff
@@ -8,3 +8,4 @@ approvers:
   - serathius
 labels:
   - sig/api-machinery
+  - sig/etcd
```

Comment:
this is different because this is still the location from which we build the etcd image.

Reply:
Ack. leaving this